packages/core/src/components/search/SearchDropdown/SearchDropdown.tsx

```diff
@@ -14,7 +14,12 @@ import { ProductSummary_ProductFragment } from '@generated/graphql'
 import SearchProductItem from 'src/components/search/SearchProductItem'
 import { formatSearchPath } from 'src/sdk/search/formatSearchPath'
 
-function SearchDropdown({ sort, ...otherProps }) {
+interface SearchDropdownProps {
+  sort: SearchState['sort']
+  [key: string]: any
```

Comment:
**question: (non-blocking)** Why is this necessary? Since we are spreading `otherProps` I couldn't understand why this is needed

Reply:
The **Typescript** needs to know the typing of all props. It was difficult for me to know what the props were because of the spread (...otherProps). So the "workaround" I had to do is that the other props can be any pair with the key string and the value any